src/typing/flow_js.ml

```diff
@@ -1262,9 +1262,6 @@ let rec flow cx (l,u) trace =
     | (_,
        (GetT(_,"toLocaleString",_) | SetT(_,"toLocaleString",_))) -> ()
 
-    | ((ObjT _ | ArrT _), GetT(reason_op,"length",tout)) ->
-      unit_flow cx (NumT.why reason_op, tout)
-
     | ((ObjT _ | ArrT _),
```

Comment:
Can you also delete this case? It was allowing you to set `.length` to anything and call `.length()` with anything. For example this is clearly wrong

    var a = { length: "duck" };
    a.length = 123;
    a.length();

and I think you should get two errors if you delete this case.

Also, sorry for not giving this feedback last time. If you're busy just let me know. I'll merge and update it myself :)